core/src/main/scala/org/http4s/otel4s/middleware/ServerMiddleware.scala

```diff
@@ -132,29 +134,34 @@ object ServerMiddleware {
           MonadCancelThrow[G].uncancelable { poll =>
             val tracerG = Tracer[F].mapK[G]
             tracerG.joinOrRoot(req.headers) {
-              tracerG.span(serverSpanName(req), init: _*).use { span =>
-                poll(f.run(req))
-                  .guaranteeCase {
-                    case Outcome.Succeeded(fa) =>
-                      span.addAttribute(Attribute("exit.case", "succeeded")) >>
-                        fa.flatMap { resp =>
-                          val out = response(resp, respHeaders) ++ additionalResponseTags(resp)
-                          span.addAttributes(out: _*)
-                        }
-                    case Outcome.Errored(e) =>
-                      span.recordException(e) >>
-                        span.addAttribute(Attribute("exit.case", "errored"))
-                    case Outcome.Canceled() =>
-                      span.addAttributes(
-                        Attribute("exit.case", "canceled"),
-                        Attribute("canceled", true),
-                        Attribute(
-                          "error",
-                          true,
-                        ), // A cancelled http is an error for the server. The connection got cut for some reason.
-                      )
-                  }
-              }
+              tracerG
+                .spanBuilder(serverSpanName(req))
+                .withSpanKind(SpanKind.Server)
+                .addAttributes(init: _*)
+                .build
+                .use { span =>
+                  poll(f.run(req))
+                    .guaranteeCase {
+                      case Outcome.Succeeded(fa) =>
+                        span.addAttribute(Attribute("exit.case", "succeeded")) >>
+                          fa.flatMap { resp =>
+                            val out = response(resp, respHeaders) ++ additionalResponseTags(resp)
+                            span.addAttributes(out: _*)
+                          }
+                      case Outcome.Errored(e) =>
+                        span.recordException(e) >>
+                          span.addAttribute(Attribute("exit.case", "errored"))
+                      case Outcome.Canceled() =>
+                        span.addAttributes(
+                          Attribute("exit.case", "canceled"),
+                          Attribute("canceled", true),
+                          Attribute(
+                            "error",
+                            true,
+                          ), // A cancelled http is an error for the server. The connection got cut for some reason.
```

Comment:
Doesn't matter, but we spell it `canceled` elsewhere, including in the Attribute name and in Cats Effect.

```suggestion
                          ), // A canceled http is an error for the server. The connection got cut for some reason.
```